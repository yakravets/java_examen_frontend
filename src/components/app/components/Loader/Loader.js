import { Spin, Space } from 'antd';
import { LoadingOutlined } from '@ant-design/icons';

export default function Loader (){
    return(
        <>
            <Space size="middle">
                <Spin size="large" indicator={<LoadingOutlined style={{ fontSize: 24 }} spin />}/>
            </Space>
        </>
    );
}