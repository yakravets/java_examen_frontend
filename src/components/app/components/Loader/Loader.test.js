import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import toJson from "enzyme-to-json";

import Loader from './Loader';

it("renders correctly", () => {
    const tree = shallow(<Loader />);
    expect(toJson(tree)).toMatchSnapshot();
});

it("renders without crashing", () => {
    shallow(<Loader />);
});