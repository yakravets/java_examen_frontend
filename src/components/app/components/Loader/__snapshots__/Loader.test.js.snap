// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <Space
    size="middle"
  >
    <Spin
      indicator={
        <ForwardRef(LoadingOutlined)
          spin={true}
          style={
            Object {
              "fontSize": 24,
            }
          }
        />
      }
      size="large"
      spinning={true}
      wrapperClassName=""
    />
  </Space>
</Fragment>
`;
